tests/fixtures/data.yml

```diff
@@ -9,12 +9,16 @@ person:
     ORIGINAL_HIRE_DATE: !!timestamp 2001-01-01 00:00:00.0
     APPOINTMENT_END_DATE: !!timestamp 2010-01-01 00:00:00.0
     JOB_TITLE: Adjunct Professor
-    PERSONNEL_SUBAREA_CODE: CFAC
-    HR_ORG_UNIT_TITLE: Chemisty
+    PERSONNEL_SUBAREA_CODE: CFAT
+    HR_ORG_UNIT_ID: "1"
+    HR_ORG_LEVEL4_NAME: Chemistry
```

Comment:
I tracked down my problem. `def test_person_feed_adds_person` and `def test_person_feed_uses_utf8_encoding` needed to have the dlc added to the record due to how the test works to match how the orcid is added. `r.update(records[0]['dlc'])` 

I didn't read the tests or errors closely before. Problem solved.